NEWS

```diff
@@ -21,6 +21,20 @@ CHANGES WITH 233 in spe
           during startup. If no such message is sent, the service now fails,
           even if the main process exited with a successful exit code.
 
+        * When units are about to be started an additional check is now done to
+          ensure that all dependencies of type BindsTo= are started, and all
+          dependencies of type Conflicts= are stopped. For this to work as
+          expected it is important to always pair up BindsTo=/Conflicts= type
+          dependencies with appropriate ordering dependencies (usually After=),
+          to ensure that the relevant requirements are fulfilled before the
+          unit is started. Previously, this was already required but did not
+          result in immediate problems as the missing ordering dependencies
+          only became visible under an unlikely race condition. With this
+          additional check this race is removed, but most BindsTo=/Conflicts=
+          dependencies that lacks the right ordering will now result in
+          failure. Hence, please check your units and look for any dependencies
```

Comment:
Sorry, I mean. Suppose I have
```
systemd-reboot.service: Job systemd-reboot.service/start finished, result=dependency
shutdown.target: Job shutdown.target/start failed with result 'dependency'.
```
What should I do to repair the machine? Is there an automatic way to fix that?